faux-mgs/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "faux-mgs"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
We haven't been bumping the version it looks like, I'm not opposed though.

Reply:
Changed to 0.1.1